riff-raff/app/controllers/authentication.scala

```diff
@@ -1,76 +1,28 @@
 package controllers
 
-import cats.data.EitherT
 import com.gu.googleauth._
 import conf.Config
 import deployment.{DeployFilter, Deployments, Record}
 import org.joda.time.DateTime
 import persistence.DataStore
-import play.api.data.Forms._
-import play.api.data._
 import play.api.i18n.I18nSupport
 import play.api.libs.json._
 import play.api.libs.ws.WSClient
 import play.api.mvc._
 import scalikejdbc._
-import utils.Json._
 import utils.LogAndSquashBehaviour
 
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext
 
 class ApiRequest[A](val apiKey: ApiKey, request: Request[A])
     extends WrappedRequest[A](request) {
   lazy val fullName = s"API:${apiKey.application}"
 }
 
-case class AuthorisationRecord(
-    email: String,
-    approvedBy: String,
-    approvedDate: DateTime
-) {
-  def contentBlob = (approvedBy, approvedDate)
-
-  def isSuperuser(config: Config) = config.auth.superusers.contains(email)
-}
-object AuthorisationRecord extends SQLSyntaxSupport[ApiKey] {
-  implicit val formats: Format[AuthorisationRecord] =
-    Json.format[AuthorisationRecord]
-
-  def apply(res: WrappedResultSet): AuthorisationRecord =
-    Json.parse(res.string(1)).as[AuthorisationRecord]
-
-  override val tableName = "auth"
-}
-
-trait AuthorisationValidator {
-  def emailDomainAllowList: List[String]
-  def emailAllowListEnabled: Boolean
-  def emailAllowListContains(email: String): Boolean
-  def isAuthorised(id: UserIdentity) = authorisationError(id).isEmpty
```

Comment:
There's more background and why we no longer need this functionality [here](https://github.com/guardian/riff-raff/pull/1464#discussion_r2301054390).